src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -152,8 +153,8 @@ export default class OrganizationDiscover extends React.Component {
   };
 
   getDateTimeFields = ({period, start, end, utc}) => ({
-    range: period || null,
-    utc,
+    range: period || (!start && !end && DEFAULT_STATS_PERIOD) || null,
```

Comment:
Why would the `onChangeTime` and `onUpdateTime` callbacks now get an invalid datetime object with falsy period, start and end values? If a default needs to be applied for URL reasons let's do it within the header component so the views that use the header don't need to know about it?